python/ray/rllib/dqn/dqn.py

```diff
@@ -355,7 +355,7 @@ class DQNAgent(Agent):
     _agent_name = "DQN"
     _default_config = DEFAULT_CONFIG
 
-    def stop(self):
+    def _stop(self):
```

Comment:
NIT/out of scope here: This should be implemented in all algorithms that have actors right? (ie, A3C and PPO)

Reply:
Hm probably. I'm not sure how reliable the ref counting is, especially if you force exit an actor.